_posts/hysteresis.md

```diff
@@ -0,0 +1,99 @@
+
+She soon came and called out my name saying, “I am so glad to see you haven’t escaped. We were all so worried about you. The other doctors called you up, and later told me something about you screaming for help! I was so worried.” I suddenly realized what was nagging me about my previous phone call to her. It was the question she had asked, “Where have you escaped and gone off to?” That seemed such a strange question to ask. Escaped? From where? And even now, whatever she just spoke seemed confusing. Just then, she approached me and I felt a prick of an injection and everything went dark. At this point, in any self-respecting story, the protagonist groggily wakes up to find himself tied to a chair. When I woke up, I found myself locked up in a room. But not tied to a chair (how disappointing). My girlfriend entered along with another man, and they both were wearing white coats. My girlfriend spoke to the man, “Thank god he did not get too far away. And he didn’t cause much damage. Good thing we found him so quickly.” The man replied, “Yes, you are right, Doctor!”. Then they both walked out and locked the door. I looked around the room and realized that it was familiar to me. I vaguely remembered having been here before. There was a bed. Upon the head of the bed was a label with my name on it. There was also a table beside it with a book lying on top of it.
+
+Sigh, the day of our marriage was the worst day of my life. It was on this day that I realized that I never was about to get married to her. It was on this day that I realized that she was my doctor in this place they call the asylum. It was on this day that I realized that she wasn’t even my girlfriend. It was on this day that I realized that I didn’t even have one. The best day of my life was the day I met my girlfriend. The worst day of my life was the day of our marriage. It just so happened that both were the same day...
+
+Oh, I am so sorry I forgot to introduce myself. They say my name is AMP. The label on the bed reads, “Acute Megalomaniac Paranoid”. I glance at the book on the table. The title reads: “All Memories Purged: Alzheimer’s Malady’s Patients”. And a subtitle in a slightly smaller font below it reads: “Anecdotal Musings Perhaps, ranging from Absent Minded Professors to Authors Mightily Prosopagnosic, causing events ranging from Amusing Marriage Pandemoniums to Amazing Medal- worthy Performances”. A book review on the cover page says: “A Master-Piece”. Hello reader, I am just Another Modern Physicist.
+
+---
```

Comment:
where is there a line at the bottom here?

Reply:
For some reason the last paragraph becomes bold without it, thats why

Reply:
Keeping the line right after text makes it a heading. Are you sure removing this line makes it bold? There's another line above somewhere (which may or may not be intentional (?))

Reply:
It does, I replaced it with "___", to not have that issue.